README.md

```diff
@@ -1,33 +1,33 @@
 # GURU-PR [![Build Status](https://travis-ci.org/guru-pr/gurupr.svg?branch=master)](https://travis-ci.org/guru-pr/gurupr) [![Code Climate](https://codeclimate.com/github/guru-pr/gurupr/badges/gpa.svg)](https://codeclimate.com/github/guru-pr/gurupr) [![Test Coverage](https://codeclimate.com/github/guru-pr/gurupr/badges/coverage.svg)](https://codeclimate.com/github/guru-pr/gurupr) [![Dependency Status](https://img.shields.io/gemnasium/guru-pr/gurupr.svg)](https://gemnasium.com/guru-pr/gurupr)
 
-Site do Grupo de Usuários Ruby do Paraná!
+Site of the Ruby users group from Paraná - Brazil!
 
-Utilizamos:
-- Github para autenticação
+We use:
+- Github for authentication
 - Heroku
 - PostgreSQL
 
-De resto, vamos desenvolvendo aos poucos!
+Otherwise, let's develop in baby steps!
 
-## Instalação normal
+## Common install
```

Comment:
Better than a translation, maybe we should make those headers more correct (I think that the ones in Portuguese could be more assertive).

There is no _normal_ installation but instead we are talking about running the project on your machine. What about `Running on your machine`?